docs/testing.rst

```diff
@@ -138,39 +140,45 @@ of the application.  To do this, we fire some requests to the login and logout
 pages with the required form data (username and password).  And because the
 login and logout pages redirect, we tell the client to `follow_redirects`.
 
-Add the following two methods to your `FlaskrTestCase` class::
+Add the following two functions to your :file:`test_flaskr.py` file::
 
-   def login(self, username, password):
-       return self.app.post('/login', data=dict(
-           username=username,
-           password=password
-       ), follow_redirects=True)
+    def login(client, username, password):
+        return client.post('/login', data=dict(
+            username=username,
+            password=password
+        ), follow_redirects=True)
 
-   def logout(self):
-       return self.app.get('/logout', follow_redirects=True)
+    def logout(client):
+        return client.get('/logout', follow_redirects=True)
 
 Now we can easily test that logging in and out works and that it fails with
-invalid credentials.  Add this new test to the class::
-
-   def test_login_logout(self):
-       rv = self.login('admin', 'default')
-       assert b'You were logged in' in rv.data
-       rv = self.logout()
-       assert b'You were logged out' in rv.data
-       rv = self.login('adminx', 'default')
-       assert b'Invalid username' in rv.data
-       rv = self.login('admin', 'defaultx')
-       assert b'Invalid password' in rv.data
+invalid credentials.  Add this new test function::
+
+    def test_login_logout(client):
+        """Make sure login and logout works"""
```

Comment:
Pedantic: PEP-257 says docstrings like this should be a sentence... aka, please end this line with a period.